spec/models/spree/payment/processing_decorator_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe Spree::Payment::Processing do
+  describe '#close!' do
```

Comment:
I think this spec should just assert that the `close!` the source receives a call to `close!` no?